src/phoenix/db/engines.py

```diff
@@ -130,6 +131,15 @@ def async_creator() -> aiosqlite.Connection:
     return engine
 
 
+def set_postgresql_search_path(schema: str) -> Callable[[Connection, Any], None]:
+    def _(connection: Connection, _: Any) -> None:
+        cursor = connection.cursor()
+        cursor.execute(f"CREATE SCHEMA IF NOT EXISTS {schema};")
+        cursor.execute(f"SET search_path TO {schema};")
+
+    return _
```

Comment:
what happens if the schema is set after you've already bootstrapped the DB?

Reply:
it'll be created as a new one: that how the integration tests work